ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFContactCollectionView)
+open class ContactCollectionView: UICollectionView {
+    struct Constants {
+        static let sideInset: CGFloat = 16.0
```

Comment:
nitpick: a better name would be "verticalInset" if this is used for both the leading and trailing edge. If it is only used for the leading edge, then "leadingInset" would be a better name.

Reply:
This is only used for the leading edge. I'll use your suggestion as it seems more appropriate!